bears/markdown/MarkdownBear.py

```diff
@@ -1,13 +1,18 @@
 import json
 
-from coalib.bearlib.abstractions.Linter import linter
+from coalib.bearlib.abstractions.Linter import linter, _create_linter
 from coalib.bears.requirements.NpmRequirement import NpmRequirement
 
 
 @linter(executable='remark',
         use_stdin=True,
         output_format='corrected',
         result_message='The text does not comply to the set style.')
+@staticmethod
```

Comment:
Um by putting this function here, the linter decorator goes to this function and not the class...